lib/js/components/Badges/Badge/Badge.vue

```diff
@@ -64,6 +102,16 @@ export default {
 				return Object.values(BADGE_COLORS).includes(value);
 			},
 		},
+		icon: {
+			type: String,
+			required: true,
```

Comment:
is it required?
How do we plan to migrate existing usages?

Reply:
Sure, it shouldn't be required